docs/.vuepress/public/assets/migrations/scripts/migrate-3.4.0.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path');
+const fs = require('fs');
+
+const ONE_RELATIONS = ['oneToOne', 'manyToOne', 'oneWay'];
+
+const createStrapiApp = async projectPath => {
+  if (!projectPath) {
+    throw new Error(`
+-> Path to strapi project is missing.
+-> Usage: node migrate-3.4.0.js [path]`);
+  }
+
+  let app;
+  try {
+    const strapi = require(require.resolve('strapi', { paths: [projectPath] }));
+    app = await strapi({ dir: projectPath }).load();
```

Comment:
You should just wrap the resolve in a try catch because if strapi fails for another reason on load it won't be displayed